llvm/test/Transforms/SimplifyCFG/X86/switch_to_lookup_table.ll

```diff
@@ -340,15 +340,23 @@ entry:
   i32 3, label %sw.bb3
   ]
 
-sw.bb: br label %if.end
-sw.bb1: br label %if.end
-sw.bb2: br label %if.end
-sw.bb3: br label %if.end
-sw.default: br label %if.end
-if.else: br label %if.end
+sw.bb:                                            ; preds = %entry, %entry
```

Comment:
Did you mean to change these lines?

Reply:
I took the chance to regenerate the test too, as the `if.else` branch was dead.